tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(apache2_bins dash_bins base-files_bin coreutils_bins adduser_bins)"
```

Comment:
Can this be shortened to just `apache2_bins dash_bins`?

And are there any tests we can add for the following?
- /usr/sbin/a2disconf
- /usr/sbin/a2dismod
- /usr/sbin/a2dissite
- /usr/sbin/a2enconf
- /usr/sbin/a2enmod
- /usr/sbin/a2ensite
- /usr/sbin/a2query